airavata-local-agent/README.md

```diff
@@ -0,0 +1,16 @@
+# Airavata UI
+
```

Comment:
Please provide a description of the local agent and what purpose it server. Also, please provide some screenshots